app/views/events/mammography/medical-information/hormone-replacement-therapy.html

```diff
@@ -75,22 +75,22 @@
     value: event.medicalInformation.hrt.hrtQuestion,
     fieldset: {
       legend: {
-        text: pageHeading,
-        classes: "nhsuk-fieldset__legend--l",
-        isPageHeading: true
+        text: "Is " + (participant | getFullName) + " currently taking HRT?",
+        classes: "nhsuk-fieldset__legend--m",
+        isPageHeading: false
```

Comment:
```suggestion
        isPageHeading: true
```
Did you mean to change this?